india_forecast_app/adjuster.py

```diff
@@ -0,0 +1,91 @@
+""" Adjuster code, adjust forecast by last 7 days of ME"""
+from datetime import datetime, timedelta
+from typing import Optional
+
+import pandas as pd
+from pvsite_datamodel.sqlmodels import ForecastSQL, ForecastValueSQL, GenerationSQL
+from sqlalchemy import INT, cast, text
+from sqlalchemy.sql import func
+
+# Wad to get all the forecast for the last 7 days made, at this time.
+# And find the ME for each forecast horizon
+
+"""
+Here is the SQL query that it based off
+
+select 
+AVG(generation.generation_power_kw - forecast_values.forecast_power_kw)
```

Comment:
Might be worth just tidying this up a bit and removing the commented out bits and the hard coded id? 

Also I think the generation and forecast order is the opposite in the example here to what is done below in the actual query

Reply:
I'll switch the order. I'll leave the harded id, just so we know what to change easily. 
For the comment outs bits, I sometimes use then, when not using the groupby, but ill add a extra comment in